dbms/src/Common/dataTypeTransform.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <Core/Types.h>
+#include <Parsers/IAST.h>
+#include <Parsers/ASTFunction.h>
+#include <Parsers/ASTIdentifier.h>
+#include <DataTypes/IDataType.h>
+#include <DataTypes/DataTypeString.h>
+#include <DataTypes/DataTypesNumber.h>
+#include <DataTypes/DataTypeDate.h>
+#include <DataTypes/DataTypeDateTime.h>
+#include <DataTypes/DataTypeNullable.h>
+#include <DataTypes/DataTypeFixedString.h>
+
+namespace DB
+{
+
+ASTPtr getDataTypeAST(const DataTypePtr & data_type);
```

Comment:
Non-descriptive function name.
Missing comments.

Reply:
(the method name was Ok before, because it was used inside single translation unit)